src/OpenSage.Game/Gui/Wnd/Elements/UIElement.cs

```diff
@@ -145,6 +145,8 @@ public bool Highlighted
             }
         }
 
+        public bool HideImage { get; set; }
```

Comment:
I'm pretty sure this is actually handled by the `W3DNoDraw` callback, as set in `MainMenu.wnd`, line 18. The only reason I didn't implement that yet is because it was nice to see the backdrop image - but now that you're starting to add the actual shell map, we can go ahead and implement `W3DNoDraw` in the `WndCallbacks` class.